core/src/main/scala/kafka/server/QuotaFactory.scala

```diff
@@ -38,7 +39,12 @@ object QuotaFactory extends Logging {
     override def isQuotaExceeded(): Boolean = false
   }
 
-  case class QuotaManagers(fetch: ClientQuotaManager, produce: ClientQuotaManager, request: ClientRequestQuotaManager, leader: ReplicationQuotaManager, follower: ReplicationQuotaManager) {
+  case class QuotaManagers(fetch: ClientQuotaManager,
+                           produce: ClientQuotaManager,
+                           request: ClientRequestQuotaManager,
+                           leader: ReplicationQuotaManager,
+                           follower: ReplicationQuotaManager,
+                           alterLogDirs: ReplicationQuotaManager) {
```

Comment:
I intentionally used alterLogDirs to reduce the length of the variable name and related config key name in other places. I think "replica" can be removed from the name because its type is `ReplicationQuotaManager`, which already includes the word "replication".

I will change it to `alteReplicaLogDirs` if you prefer. What do you think?